Assets/Scripts/Game/Game.cs

```diff
@@ -60,28 +60,47 @@ public void PlaceTile(int tileID, int q, int r, int marketIndex = 0)
 		}
 	}
 
-	public void DoubleTile(Tile tile, int q, int r)
+	public void DoubleTile(int tileID, int q, int r)
 	{
+		Tile tile = pack.GetMetadata()[tileID];
+
 		// Check if the player has enough money
 		if (activePlayer.money < tile.price)
 		{
+			PopupManager.Show ("Not enough money.");
 			throw new InvalidOperationException();
```

Comment:
You should return instead of throwing an exception; otherwise execution of code following an invoke of this method will not run.